plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1853,6 +1848,69 @@ public Optional<ConnectorOutputMetadata> finishInsert(
         return Optional.empty();
     }
 
+    private long getInsertCommitVersion(
```

Comment:
Is it possible to add another commit for extracting this method without retrying? It's a little hard to review the diff.